gerrymandering/gerrymandering.ipynb

```diff
@@ -115,11 +162,11 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "x_range, y_range = ((-13884029.0, -7453303.5), (2818291.5, 6335972.0)) # Continental USA\n",
-    "shade_defaults = dict(x_range=x_range, y_range=y_range, x_sampling=10, y_sampling=10, width=1200, height=682,\n",
-    "                      color_key=color_key, aggregator=ds.count_cat('race'),)\n",
-    "shaded = datashade(census_points, **shade_defaults)\n",
-    "shaded"
+    "# x_range, y_range = ((-13884029.0, -7453303.5), (2818291.5, 6335972.0)) # Continental USA\n",
+    "# shade_defaults = dict(x_range=x_range, y_range=y_range, x_sampling=10, y_sampling=10, width=1200, height=682,\n",
+    "#                       color_key=color_key, aggregator=ds.count_cat('race'),)\n",
+    "# shaded = datashade(census_points, **shade_defaults)\n",
+    "# shaded"
```

Comment:
I'm not sure what the commented-out lines are meant to convey?

Reply:
This notebook is meant to help @jtao1 getting started on updating the code of this project. I think it's easier to understand if the old code is kept around.